src/components/pages/Landing/RenderLandingPage.jsx

```diff
@@ -1,11 +1,132 @@
 import React from 'react';
+import WhiteLogo from '../../../styles/Images/WhiteLogo.png';
+import GrantRatesByOfficeImg from '../../../styles/Images/bar-graph-no-text.png';
+import GrantRatesByNationalityImg from '../../../styles/Images/pie-chart-no-text.png';
+import GrantRatesOverTimeImg from '../../../styles/Images/line-graph-no-text.png';
+import HrfPhoto from '../../../styles/Images/HRF2.png';
```

Comment:
This image could be better optimized. 
Currently : 3.9 mb (.png)
Expected : There is no transparency layer needed here. It could be a compressed .jpeg.